codechecker_lib/analyzers/result_handler_plist_to_db.py

```diff
@@ -33,6 +33,127 @@ def __init__(self, buildaction, workspace, run_id):
         super(PlistToDB, self).__init__(buildaction, workspace)
         self.__run_id = run_id
 
+    def __store_bugs(self, files, bugs, connection, analisys_id):
+        file_ids = {}
+        # Send content of file to the server if needed
+        for file_name in files:
+            file_descriptor = connection.need_file_content(self.__run_id,
+                                                           file_name)
+            file_ids[file_name] = file_descriptor.fileId
+            if file_descriptor.needed:
```

Comment:
This part should be hardened. What if the original source file is not available. It might be still useful to store the results without the source file content. Viewing/filtering the result list for statistics could be possible, just viewing the file content with the bug will fail.  Further improvement might be needed for the web GUI #320